src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,239 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+/**
+* Represents the static data of a user.
+*/
+public class User {
+    private final String userID;
+    private final String username;
+    private final String firstName;
+    private final String lastName;
+    private final String profilePicture;
+    private final ArrayList<String> gamesCreated;
+    private final int numGamesFinished;
+    private final int numGamesCreated;
+    private final ArrayList<Pair<String, Long>> gamesCompletedWithTime;
+
+    /**
+    * Builder class used to create User objects.
+    */
+    public static class Builder {
+        // Required parameters
+        private final String userID;
+
+        // Optional parameters - initialized to default values
+        private String username = null;
```

Comment:
You don't need to initialize "username", "firstName", "lastName", "profilePicture" to null. The following code does the same thing by initializing them to null.

private String username;
private String firstName;
private String lastName;
private String profilePicture;

Reply:
I just like the style of this better because I'm setting all the ints to -1, so I would like to set the Strings to some value as well. I think looks weird to set some variables equal to some initial value, while leaving the others untouched, even though they get set to null by default.